packages/salesforcedx-vscode-core/test/vscode-integration/util/authInfo.test.ts

```diff
@@ -80,14 +91,17 @@ describe('OrgAuthInfo', () => {
       );
       expect(showMessageStub.calledOnce).to.equal(true);
 
-      configUtilStub.restore();
+      configAggregatorStub.restore();
```

Comment:
you should never have to do this with a sandbox.  I'd just add an `afterEach` to the top level describe that does a `sandbox.restore()`.  If we have tests that depend on stubs across tests those shouldn't be happening in an individual test.

Reply:
looks like there already is sandbox.restore in afterEach.  Ill remove these